lib/data_hygiene/bulk_organisation_updater.rb

```diff
@@ -0,0 +1,68 @@
+require "csv"
+
+module DataHygiene
+  class BulkOrganisationUpdater
+    def initialize(filename)
+      @filename = filename
+      @all_organisations = Queries::GetLinkables.new(document_type: "organisation").call
+    end
+
+    def call
+      CSV.foreach(filename, liberal_parsing: true, headers: :first_row) do |row|
+        process_row(row)
+      end
+    end
+
+    def self.call(*args)
+      new(*args).call
+    end
+
+  private
+
+    attr_reader :filename
+
+    def process_row(row)
+      document = find_document(row)
+      return unless document
+
+      organisations = find_organisations(row)
+      update_document(document, organisations)
+
+      puts "Sucessfully updated"
```

Comment:
How about including something identifiable, so we know which row has successfully updated?

```suggestion
      path = row.fetch("Path")&.strip
      puts "Sucessfully updated: #{path}"
```